### ISSUE TITLE
Unable to install fork CLI

### ISSUE THREAD

**Issue by Aircl0wn** [ForkIssues/Tracker#999]
After reinstalling fork on a fresh macbook, clicking the 'install fork CLI' button does nothing.

Any way for me to provide more information?

```
ℹ️[IntegrationPreferencesController.swift:369] install() > Installing cli tool...
❗️[IntegrationPreferencesController.swift:371] install() > Can't install cli tool. Error Dictionary: '["NSAppleScriptErrorRange": NSRange: {0, 163}, "NSAppleScriptErrorNumber": 1, "NSAppleScriptErrorAppName": Fork, "NSAppleScriptErrorMessage": mkdir: /usr/local/bin: Permission denied^Mln: /usr/local/bin/fork: No such file or directory, "NSAppleScriptErrorBriefMessage": mkdir: /usr/local/bin: Permission denied^Mln: /usr/local/bin/fork: No such file or directory]'
```
log says access denied.
I would assume it would give a password notification but this doesn't happen.

**Comment by uvw** [comment]
I guess Fork (as some other dev apps on macOS) assumes that Homebrew is installed on our MacBooks, and the current user already has RW permissions at `/usr/local/bin`.

As a fix, I would suggest changing Fork to put CLI into `/Users/$USER/.local/bin` instead. I noticed that many apps are following this approach, CLIs for all JetBrains IDEs and a few K8s tools for instance.

**Comment by Aircl0wn** [comment]
This seems to be correct, as brew wasn't setup yet when i tried this.

**Comment by samdeane** [comment]
Yep, I had a similar problem, and realised that `/usr/local/bin` didn't exist.
I created it and then the Install button in Fork worked.

**Comment by richardbowden** [comment]
> After reinstalling fork on a fresh macbook, clicking the 'install fork CLI' button does nothing.
> 
> Any way for me to provide more information?
> 
> ```
> ℹ️[IntegrationPreferencesController.swift:369] install() > Installing cli tool...
> ❗️[IntegrationPreferencesController.swift:371] install() > Can't install cli tool. Error Dictionary: '["NSAppleScriptErrorRange": NSRange: {0, 163}, "NSAppleScriptErrorNumber": 1, "NSAppleScriptErrorAppName": Fork, "NSAppleScriptErrorMessage": mkdir: /usr/local/bin: Permission denied^Mln: /usr/local/bin/fork: No such file or directory, "NSAppleScriptErrorBriefMessage": mkdir: /usr/local/bin: Permission denied^Mln: /usr/local/bin/fork: No such file or directory]'
> ```
> 
> log says access denied. I would assume it would give a password notification but this doesn't happen.

this feels like a very sensible default setting. and easier to create if the path does not exist
